geometry_msgs/Point point
std_msgs/Bool return_ground_truth
---
gazebo_msgs/ModelState ground_truth
turtlebot_ctrl/TurtleBotScan scan_data